src/components/skeleton/ApplicationsMenu.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="text-center">
+    <v-menu :nudge-width="200" open-on-hover bottom offset-y >
+      <template v-slot:activator="{ on, attrs }">
+       <v-btn dark v-bind="attrs" text v-on="on">
+          {{ $t('softwares') }}
+          <v-icon right>mdi-menu-down</v-icon>
+        </v-btn>
+      </template>
+
+      <v-card class="py-4">
+        <v-list-item
+          v-for="application in applications"
+          :key="application.name"
+          link
```

Comment:
You forgot to implement the link to the application

```suggestion
          :href="application.link"
          link
```